Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -34,18 +33,16 @@
 
 TEST_DATA_INTEGRATION_YML_REGEX = r'Tests\/scripts\/infrastructure_tests\/tests_data\/mock_integrations\/.*\.yml'
 INTEGRATION_REGEXES = [
-    INTEGRATION_REGEX,
-    BETA_INTEGRATION_REGEX,
-    PACKS_INTEGRATION_REGEX,
+    PACKS_INTEGRATION_PY_REGEX,
+    PACKS_INTEGRATION_PS_TEST_REGEX,
```

Comment:
where is this being used? looks like a weird code change
add a comment for what this list should contain, (files to ignore,  test files, etc...)

Reply:
because we use demisto-sdk from master (which includes changes in the constants) and not from the release, it's a pull from Anar's PR of adjustments for the sdk changes. when the release is out, it will be reverted.